fundamentals/impl.tex

```diff
@@ -31,7 +31,9 @@ \subsection{Modular Software Architecture}
 
 The architecture of \Cyclus allows developers 
 to define nuclear fuel cycle agents and processes independent of the simulation 
-logic. This encapsulated `plug-in'
+logic. To acheive this, agents are developed using the \Cyclus framework \gls{API}, a set of functions and protocols
+which assist in agent development and specify how they should be defined. 
+This encapsulated `plug-in'
```

Comment:
change to:
'that assist in agent development and specify how agents should be defined.'
Also, 'achieve' is misspelled in line 34.
Otherwise, this clarification addresses my concerns.